documentation/src/docs/controller-service.md

```diff
@@ -154,296 +151,276 @@ Controller Process Diagram
 
 There are two ports exposed by Controller: **Client-Controller APIs** and
 **Administration APIs**. The client Controller communication is implemented as `RPC` which
-exposes APIs to perform all stream related control plane operations.
-Apart from this Controller also exposes an administrative API set
-implemented as `REST`.
+exposes APIs to perform all Stream related control plane operations.
+Apart from this Controller also exposes an administrative APIs implemented as `REST`.
 
-Each endpoint performs appropriate call to the *Controller Service backend subsystem*
+Each endpoint performs appropriate call to the Pravega Controller Service *backend subsystem*
 which has the actual implementation for various operations like create, read, update and
 delete (CRUD) on entities owned and managed by Controller.
 
 ### gRPC  
 
-Client Controller communication endpoint is implemented as a [gRPC](https://grpc.io/)
-interface. Please check the complete list of [APIs](https://github.com/pravega/pravega/blob/master/shared/Controller-api/src/main/proto/Controller.proto).
-This exposes APIs used by Pravega clients (readers, writers and stream
-manager) and enables stream management. Requests enabled by this API
-include creating, modifying, and deleting streams.
-The underlying `gRPC` framework provides both _synchronous_ and _asynchronous_ programming models.
+Client Controller communication endpoint is implemented as a [`gRPC`](https://grpc.io/)
+interface. Please check the complete list of [APIs](https://github.com/pravega/pravega/blob/master/shared/controller-api/src/main/proto/Controller.proto). This exposes APIs used by Pravega clients (Readers, Writers and Stream
+Manager) and enables Stream management. Requests enabled by this API
+include *creating, modifying,* and *deleting* Streams.
+The underlying `gRPC` framework provides both **_synchronous_** and **_asynchronous_** programming models.
 We use the asynchronous model in our client Controller interactions so that the client thread does not block on the response from the server.  
-To be able to append to and read data from streams, writers and readers
-query Controller to get active segment sets, successor and predecessor
-segments while working with a stream. For transactions, the client uses
-specific API calls to request Controller to create and commit
-transactions.
+
+To be able to append to and read data from Streams, Writers and Readers
+query Controller to get active Stream Segment sets, successor and predecessor
+Stream Segments while working with a Stream. For Transactions, the client uses
+specific API calls to request Controller to _create_ and _commit_
```

Comment:
We are missing _abort_ transactions, right?

Reply:
@RaulGracia https://github.com/pravega/pravega/blob/d618cdf5d4aa7890d0bfa6e804aaa5fb60c74167/documentation/src/docs/controller-service.md#abort-transaction

Reply:
@RaulGracia Abort and ping included!